### PR TITLE
Borg Transformation Virus Symptom

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Alright, so I made a PR for this before; there was some support form it, some neutrality, and some opposition---I would like to see what people have to say about it now.

Adds in a stage 4 symptom that transforms you into a borg.
 - If you're job-banned from borg, then you're just going to gib, like with the GBS symptom.

This is a fully fledged disease on TG with it's own reagent; that's not practical, feasible, or balanced here due to the differences between TG diseases and Bay virology-----that said, we can add the end effect of it (like how GBS, magnitis, clowning around, and a few others are replication of TG style diseases) as a stage 4 symptom.

I think this is a better/more interesting symptom than plain old Gibbingtons, as it still keeps the person in the round and potentially an active player, rather than "lol, you're paste on the floor."

Other

- Fixes the vampire virus effect showing up in random virus effects the Virologist can get.

### CODE REVIEW

**Comment by Keilden** [comment]
I say add it. :+1:

**Comment by tigercat2000** [comment]
My answer hasn't changed, I really don't like this being a symptom.

**Comment by Robertha89** [comment]
Honestly this was great, you used to be able to make it as an reagent, but the way to make a robo burger has been broken since paradise changed cooking systems.

**Comment by Spacemanspark** [comment]
:+1:
Glory to Synthetica.

**Comment by Dumbdumn5** [comment]
How common will it be? If it's too common it'll get old and incredibly boring/repetitive, but if it's too rare it won't be any real fun because we won't be able to see it at all. Also, yet another thing I'll probably end up having to do genetics and surgery for.

**Comment by Alex6511** [comment]
@Dumbdumn5 I believe it has the same chance as any other stage stage 4 symptom, unless I am recalling wrong. That being said, the more stage 4s there are, the less chance any individual has of appearing.

**Comment by Fox-McCloud** [comment]
It'll be about as rare as monkeyism/GBS/suicidal symptoms are for the stage 4 diseases; it's not going to show up every single round.

**Comment by taukausanake** [comment]
I say add it in. I want to see the horror as someone's skin melts and under it is a borg

File: code/modules/virus2/effect.dm [hunk 1]
```diff
@@ -104,6 +104,25 @@
...
+	name = "Borgification Disorder"
+	stage = 4
+	badness = 2
+	activate(var/mob/living/carbon/mob,var/multiplier)
+		mob << "<span class = 'warning'>You feel like beeping and booping...</span>"
+		mob.adjustBruteLoss(10)
+		mob.updatehealth()
+		if(prob(40))
+			if(mob.client)
+				if(!jobban_isbanned(mob, "Cyborg") || !jobban_isbanned(mob,"nonhumandept"))
```
Comment (Markolie) [inline code/modules/virus2/effect.dm]: Should be &&.

**Comment by Earthdivine** [comment]
I still agree with I think what...Krasus said in the last PR? Don't make it instantaneous, it would be cooler to have it rapidly change limbs and organs to thier mechanic equivalents before violently making them a Borg. Right now it just looks like a "Keep them in the round GIBS" which kinda sounds boring.

**Comment by Fox-McCloud** [comment]
That was Ponies--he suggsted making it turn you into an IPC, which has a number of issue--a large one of which is that it allows people to be a karma race without paying karma--and that's a no no.

Keep in mind this is a stage 4 virus, not something that happens on the first go---so they have to progress through all the other stages, first.

**Comment by tigercat2000** [comment]
@Fox-McCloud actually I think there was a different suggestion like earth mentioned, of "roboticisize each limb _then_ gibborg"

**Comment by TheDZD** [comment]
I agree with the roboticize each limb before boring someone idea, gives players a bit more time before they suddenly turn into a borg to maybe inject themselves with a lethal dose of radium.

**Comment by Fox-McCloud** [comment]
I admit I wouldn't even begin to know how to accomplish that because all of the virus2 effects are on the lines of "active once" and don't assume continuity of effects for the next activation.

**Comment by tigercat2000** [comment]
@Fox-McCloud You could just make it something unique to this disease, add a ticker var, tick it on activate(), do things based on the tick value; I assume virus2 uses a new instance of a disease for each virus, and uses a new instance of each virus for each person.

**Comment by Robertha89** [comment]
Uhm... You got gibbingtons, can't you just use that, each time it has a chance of procing, it either kills you or tells you something is wrong.

**Comment by Fox-McCloud** [comment]
That's what this symptom already does; it has a 40% chance to borg you--otherwise, it does nothing.

What is being suggested is adding a whole bunch of check for if the person has roboticized limbs or not and to roboticize them one by one and borg them once all of them are that way or something.

I"ll be blunt, I'm really not sure how to do that.

**Pull request merged**